v2/emailpassword/nestjs/guide.mdx

```diff
@@ -414,11 +400,16 @@ import { Session } from './auth/session/session.decorator';
 @Controller()
 export class AppController {
   // ...
-  @Get('test')
+  @Get('/sessioninfo')
   @UseGuards(new AuthGuard())
-  async getTest(@Session() session: SessionContainer): Promise<string> {
-    // TODO: magic
-    return "magic";
+  getSessionInfo(
+    @Session() session: SessionContainer,
```

Comment:
the auth guard modifies the req object to add the session prop to it. How is that automatically picked up here?

Also, call this a test API instead of sessioninfo cause sometimes people think this is a necessary apis thats needed for supertokens integration

Reply:
Do we want to do the same thing in the cli example(s)? We could also add comments both here and there to explain that these are just example/tests endpoints.
The session prop is picked up by adding the Session decorator (defined elsewhere in this guide)

Reply:
Changed it to `/test` and added a comment.

@porcellus  I didn't make this change in the CLI because the frontend code expects the `/sessioninfo` endpoint there.